pkg/faketikv/initializer.go

```diff
@@ -13,20 +13,129 @@
 
 package faketikv
 
-// Initializer defines an Init interface
-// we can implement different case to initialize cluster
+import (
+	"context"
+	"fmt"
+
+	"github.com/BurntSushi/toml"
+	log "github.com/Sirupsen/logrus"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/pd/server/core"
+)
+
+type localAlloc struct {
+	id uint64
+}
+
+func (l *localAlloc) AllocID() (uint64, error) {
+	l.id++
+	return l.id, nil
+}
+
+// Initializer defines an Init interface.
+// we can implement different case to initialize cluster.
 type Initializer interface {
 	Init(args ...string) *ClusterInfo
 }
 
-// TiltCase will initialize cluster with all regions distributed in 3 nodes
+// TiltCase will initailize cluster with all regions distributed in 3 node.
 type TiltCase struct {
 	NodeNumber   int `toml:"node-number" json:"node-number"`
 	RegionNumber int `toml:"region-number" json:"region-number"`
+	alloc        *localAlloc
 }
 
-// NewTiltCase returns tiltCase
-func NewTiltCase() *TiltCase {}
+// NewTiltCase returns tiltCase.
+func NewTiltCase() *TiltCase {
+	return &TiltCase{alloc: &localAlloc{}}
+}
 
-// Init implement Initializer
-func (c *TiltCase) Init(addr string, args ...string) *ClusterInfo {}
+// Init implement Initializer.
+func (c *TiltCase) Init(addr string, args ...string) *ClusterInfo {
+	path := args[0]
+	err := c.parser(path)
+	if err != nil {
+		log.Fatal("initalize failed: ", err)
+	}
+	nodes := make(map[uint64]*Node)
+	regions := core.NewRegionsInfo()
+	var ids []uint64
+	for i := 0; i < c.NodeNumber; i++ {
+		id, err1 := c.alloc.AllocID()
+		if err1 != nil {
+			log.Fatal("alloc failed", err1)
+		}
+		node, err1 := NewNode(id, fmt.Sprintf("mock://tikv-%d", id), addr)
+		if err != nil {
+			log.Fatal("New node failed", err1)
+		}
+		nodes[id] = node
+		if len(ids) < 3 {
```

Comment:
why?

Reply:
I use the ids to generate region, all regions will distribute in the three nodes.